landing-pages/sudoku-for-seniors/components/PayPerCrawlContent.tsx

```
@@ -0,0 +1,130 @@
+        {isPayingCrawler && bookData && (
+          <>
+            <meta name="ai:access_level" content={paymentTier} />
+            <meta name="ai:content_type" content="puzzle_book" />
+            <meta name="ai:puzzle_count" content={String(bookData.puzzleCount)} />
+            <meta name="ai:difficulty" content={bookData.difficulty} />
+            <meta name="ai:solutions_included" content="true" />
+            
+            {/* Rich structured data for paying crawlers */}
+            <script
+              type="application/ld+json"
+              dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)